[Blog] OSM blog post

chandrashritii opened the pull request:
**Description**

This PR adds the OSM announcement post to the Layer5 blog.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

leecalcote commented:
Removed a couple files.

leecalcote commented on collections/_blogs/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh.md:
```diff
@@ -0,0 +1,30 @@
+---
+<h2 class="center" style="color:#666;font-size:1.25em;"> Adding another service mesh to the landscape and another adapter to Meshery</h2>
+
+[Open Service Mesh (OSM)](https://openservicemesh.io/) was released on August 5th by Microsoft. <img class="image-right" src="/assets/images/posts/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh/meshery-adapter-for-open-service-mesh.png" />Boasting some highly recommendable features, including but not limited to being open-source, lightweight, and SMI conformant, OSM is a welcome  ...
```
We'll want to shoot for less opinion, less excitement about OSM, specifically. We do want to be positive about the service meshes, but we're service mesh neutral. Can we think of something other than "meshiness"?

I can help, if I get back to this before you do.

leecalcote commented on collections/_blogs/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh.md:
```diff
@@ -0,0 +1,30 @@
+<h2 class="center" style="color:#666;font-size:1.25em;"> Adding another service mesh to the landscape and another adapter to Meshery</h2>
+
+[Open Service Mesh (OSM)](https://openservicemesh.io/) was released on August 5th by Microsoft. <img class="image-right" src="/assets/images/posts/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh/meshery-adapter-for-open-service-mesh.png" />Boasting some highly recommendable features, including but not limited to being open-source, lightweight, and SMI conformant, OSM is a welcome  ...
+With its own control plane and a data plane built by exacting Envoy APIs, OSM is aimed to be a holistic, community-led approach to service meshes, that can be seen as a direct implementation of the SMI specification. 
```
Not community-led. Entirely closed source until they opened it up right at the moment they can announce it.

leecalcote commented on collections/_blogs/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh.md:
```diff
@@ -0,0 +1,30 @@
+
+[Open Service Mesh (OSM)](https://openservicemesh.io/) was released on August 5th by Microsoft. <img class="image-right" src="/assets/images/posts/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh/meshery-adapter-for-open-service-mesh.png" />Boasting some highly recommendable features, including but not limited to being open-source, lightweight, and SMI conformant, OSM is a welcome  ...
+With its own control plane and a data plane built by exacting Envoy APIs, OSM is aimed to be a holistic, community-led approach to service meshes, that can be seen as a direct implementation of the SMI specification. 
+Drawing lessons from existing service mesh projects, the central goal is providing an effortless way for Kubernetes operators to install, maintain and run service meshes without having to make the seemingly difficult choice of choosing one from the long list of available options in the market.
```
drawing lessons and code from other service meshes

leecalcote commented on collections/_blogs/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh.md:
```diff
@@ -0,0 +1,30 @@
+---
```
Changed filename. The link to the blog post was broken otherwise.

leecalcote commented on collections/_blogs/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh.md:
```diff
@@ -0,0 +1,30 @@
+layout: post
+title:  "Announcing the Meshery Adapter for Open Service Mesh"
+date: 2020-09-04 10:17:05 +0000
+image: /assets/images/posts/2020-09-05-announcing-the-meshery-adapter-for-open-service-mesh/meshery-adapter-for-open-service-mesh.png
```
OSM is pixelated this image. You can use the svg in this repo to avoid pixelation.

Try to remove some of the surrounding whitespace on the sides and top of the image, if you would. (I can do this tomorrow).

chandrashritii commented:
@leecalcote 
- Fixed the language to make it sound service mesh neutral
- Made suggested changes
- Removed whitespace around the image and converted the osm logo to svg
